src/Microsoft.SqlTools.Dmp.Contracts/ProviderDetails.cs

```diff
@@ -0,0 +1,21 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.Dmp.Contracts
```

Comment:
>Dmp [](start = 29, length = 3)

I'd prefer the have Dmp expanded out to DataProtocol or something...though I can see that might make some of the long names event longer.

Reply:
I'll let you fight with @kevcunnane on this... He told me what I originally called it "Microsoft.SqlTools.DataManagementPrototcol.Contracts" was so long no one would use it. So I spent a few hours to rename everything, lol.

Reply:
Maybe DataProtocol is the acceptable compromise. I don't love super-long namespaces but that's 1 word less than the original 😃